examples/retrieve_avatar.py

```diff
@@ -74,14 +74,14 @@ def run_simple_example(self):
             self.remote_jid
         )
 
-        for metadatum in metadata["image/png"]:
-            if metadatum.has_image_data_in_pubsub:
+        for metadatum in metadata:
+            if metadatum.can_get_image_bytes_via_xmpp:
                 image = yield from metadatum.get_image_bytes()
                 with open(self.output_file, "wb") as avatar_image:
                     avatar_image.write(image)
                 return
 
-        print("retrieving avatar failed: no avatar in pubsub")
+        print("retrieving avatar failed: no png-avatar in pubsub")
```

Comment:
Does this code check for PNG? (The XEP *says* that the PubSub data avatars must be PNG, but …)

Reply:
Ow, this is a left-over from an earlier version, where I tested for the mime-type, I stopped this due to the possibility of `mime_type is None`. It should not even say pubsub, as it falls back to vCard.